verify_examples.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
I'd say just use bash and let's not pretend we care to be ran under every possible implementation of POSIX shell.